Source/MLX/ParameterTypes.swift

```diff
@@ -35,6 +35,10 @@ public struct IntOrPair: ExpressibleByIntegerLiteral, ExpressibleByArrayLiteral
     public init(_ values: (Int, Int)) {
         self.values = values
     }
+
+    public init(_ value: Int) {
+        self.values = (value, value)
+    }
```

Comment:
`IntOrPair` (and friends) worked fine for literals but there was no easy way to construct them from variables, e.g. `configuration.convolutionInKernel`